j2k/src/org/jetbrains/kotlin/j2k/Converter.kt

```diff
@@ -424,6 +425,24 @@ class Converter private constructor(
         }
     }
 
+
+    private fun specialAnnotationPropertyCases(field: PsiField): Annotations {
+        val javaSerializableInterface = PsiType.getTypeByName("java.io.Serializable", project, GlobalSearchScope.allScope(project)).resolve()
```

Comment:
And also `JavaPsiFacade.findClass()` instead of `PsiType.geTypeByName()`. And `CommonClassNames.JAVA_IO_SERIALIZABLE` instead of a hard-coded constant.